lib/koans/10_enums.ex

```diff
@@ -37,26 +43,31 @@ defmodule Enums do
     assert Enum.member?([1,2,3], 30) == :__
   end
 
+  def multiply_by_ten(n), do: 10 * n
+
   koan "map converts each element of a list by running some function with it" do
-    assert Enum.map([1,2,3], fn element -> element * 10 end) == :__
+    assert Enum.map([1,2,3], &multiply_by_ten/1) == :__
   end
 
   koan "You can even return a list with entirely different types" do
-    assert Enum.map([1,2,3], fn element -> rem(element, 2) == 0 end) == :__
+    assert Enum.map([1,2,3], &is_even?/1) == :__
   end
 
+
   koan "But keep in mind that the original list remains unchanged" do
-    input = [1,2,3]
-    assert Enum.map(input, fn element -> rem(element, 2) == 0 end) == [false, true, false]
+    input = [1,2,3,4]
+    assert Enum.map(input, &is_even?/1) == :__
     assert input == :__
   end
 
+  def is_odd?(n), do: rem(n, 2) == 1
```

Comment:
I find it weird because there are quite a lot of functions in Elixir stdlib that look like `is_list` without the question mark. I'd still follow the styleguide though...